railties/test/application/bin_setup_test.rb

```diff
@@ -43,18 +43,19 @@ def test_bin_setup_output
 
         # Ignore line that's only output by Bundler < 1.14
         output.sub!(/^Resolving dependencies\.\.\.\n/, "")
+        output.gsub!(/^The dependency .* will be unused .*\.\n/, "")
```

Comment:
Maybe we could add a different comment here? The comment above make it should like this line is for bundler < 1.14

Reply:
Ah, good catch. I'll fix it!